common/src/main/java/com/wynntils/core/chat/tabs/ChatTabModel.java

```diff
@@ -60,13 +61,18 @@ public static void addMessageToTab(ChatTab tab, Component message) {
     }
 
     public static void setFocusedTab(ChatTab focused) {
+        if (Objects.equals(focusedTab, focused)) {
```

Comment:
Again, I didn't provide info about this PR. With the implemented equals method and this check, we do not overwrite the current focused if it is the same as the new one.